### ISSUE TITLE
Align handling of alttext in mathajx-node and core mathjax

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-node#281]
This is a continuation of #207. 

We jump through a few hoops to put the (dummy) speech text onto the output. It seems to me this piece could simply move up to core mathjax.

E.g., mathjax-node would simply push the input source to `root.alttext` and leave the rendering (to aria-label or title tag) to MathJax. 

We could also move it entirely upstream, i.e., the idea of "use TeX/ascii as alttext".

**Comment by dpvc** [comment]
Either of those sound reasonable.  Probably providing an option on the input jax to use the original as alttext would be good.

**Comment by pkra** [comment]
Cleaning up my old issues.
